doc/sphinx-guides/source/installation/prerequisites.rst

```diff
@@ -292,6 +273,21 @@ If the installed location of the convert executable is different from ``/usr/bin
 
 (see the :doc:`config` section for more information on the JVM options)
 
+Installing maven
```

Comment:
to be consistent with header names it should be probably be: "Maven" instead of "Installing maven"